src/vcpkg/commands.autocomplete.cpp

```diff
@@ -160,9 +160,9 @@ namespace vcpkg
                 if (Strings::case_insensitive_ascii_equals(metadata->name, "install") && results.size() == 1 &&
                     !is_option)
                 {
-                    const auto port_at_each_triplet =
+                    auto port_at_each_triplet =
                         combine_port_with_triplets(results[0], paths.get_triplet_db().available_triplets);
-                    Util::Vectors::append(&results, port_at_each_triplet);
+                    std::move(port_at_each_triplet.begin(), port_at_each_triplet.end(), std::back_inserter(results));
```

Comment:
This is actively worse than before, since you've changed a bulk insert into a one at a time insert.